web/src/containers/BudgetSummary.js

```diff
@@ -156,12 +173,58 @@ HeaderRow.propTypes = {
   years: PropTypes.array.isRequired
 };
 
+const FrozenHeaderCol = ({title}) => (
+  <tr>
+      <td className="bold">
+        {title}
+      </td>
+  </tr>
+);
+
+FrozenHeaderCol.propTypes = {
+  title: PropTypes.string.isRequired
+};
+
 const BudgetSummary = ({ activities, data, years }) => (
-  <div className="overflow-auto">
-    <table className="table-cms table-fixed" style={{ minWidth: 1000 }}>
+  <div className="table-frozen-col-wrapper">
+    <div className="table-frozen-col-scroll-window">
+      <table className="table-cms table-frozen-col-pane" aria-hidden="true">
```

Comment:
@mgwalker I'd like your take on this -- basically, I'm creating a secondary table that is just the "header" cells on the left. I don't love this approach, but nothing else I've tried works reliably cross-browser. I think I could abstract it out further than it currently is so that implementing scrolly tables across the app is easier, but I wanted to get your feedback before I go after :all-the-things:

Reply:
Hmm.  My first reaction is "that's cool!"  But then I wonder how you make sure the header cells are the same height as the content cells in the same row if they're in a different table.  I guess we could explicitly set cell heights, which doesn't seem like it should be a problem?

An alternative could be to set all cell heights and also set the header cell _widths_, and then make the header cells absolutely positioned.  Playing with it in the devtools suggests it works but it might create some quirks with padding.

Reply:
So I'm making a couple of assumptions here: one is that the content in all of these tables is budgetary, and thus will only be one line. This seems to be true so far, but I don't know if it will always be in the future. By setting the width of that first column explicitly on both tables, I can make sure the tallest element (which is the first column's cells) is the same in each.

The other way to do this is to actually create a duplicate of the _entire_ table, then set `visibility: hidden` on all of the cells except the first column. That will size the columns exactly the same in both tables, but will also, y'know, create an entire duplicate table. 

So... there isn't a _great_ way to do this that keeps the table in semantic markup. If you want me to go the clone-the-whole-table route for stability, I'm cool with that. (There are some slicker solutions out there, but they all involve making a "table" out of divs, which is going to be pretty impossible for a screen reader to parse.)

Reply:
Oh, that makes sense re: assuming the rows will be one line.  I think it's all clicking in my head now.

Gosh having a whole duplicate table sounds pretty rough.  I think I'd vote against it, but I _*am*_ curious - would having an invisible table over the top prevent scrolling?  (I assume the top table would not enable scrolling, in order to get the header column to be fixed, but 🤷‍♂️.)  Anyway...

Okay, I gotcha now.  I was just seeing a quirk locally where the content cells weren't as tall as the header cells, but it looks like that's going to get whipped.  I think this is a sensible approach.  Feels a lot less messy than a duplicate table, too.  😬

Reply:
With having a duplicate table, what happens to that fancy row that expands with the calculation inside it? I noticed that left in the recent changes that got pushed, but not sure if it's on purpose or an accidental casualty 😬 

![screen shot 2018-11-08 at 12 40 30 pm](https://user-images.githubusercontent.com/11636908/48216706-6f115e00-e353-11e8-9d82-f8861db5df56.png)

Reply:
The duplicate table approach wasn't one I'd seen before, and it kinda blew my mind. (yes, you can still scroll the underlying table! wat) Here's an example: https://codepen.io/paulobrien/pen/gWoVzN

I'd be interested in seeing that bug you saw locally, just to make sure I'm not wrong about any of my assumptions. 😬 But I'll polish this up, then, and ping you for review when it's done. Thanks for the sanity check!

Reply:
@jenniferthibault Oops, I missed this earlier -- the code to make this work is still in the table, but I couldn't find anywhere it had been used? @mgwalker, do you know?

Reply:
(also, I should note I've never seen said button in action -- I'd love it if someone could give me a demo!

Reply:
You trigger it by clicking the teensy plus sign on the right side of the cell.  I'm actually not super familiar with how it works, but I notice it seems to be really slow and not amazingly formatted.  😛

It's the `<DataRowDetails>` component that's used inside the `<DataRow>` in the `BudgetSummary.js` file.

Reply:
I missed it in my design audit (fixed now) because the pattern only appears _once you put data into the tables above_.

Reply:
ooooooh. Hrm. Okay. I'm going to have to think about this and play with it a bit.